MH-FiorellaTheme5/source/partials/_body-message.erb

```diff
@@ -0,0 +1,14 @@
+<section class="body-message">
+  <div class="title-message">
+    <h2 class="title">
+        <span>ALWAYS FRESH</span>
```

Comment:
Why are you inserting a span inside a header?

Reply:
@IsmaelVillavicencio did this correction on his branch.... Do I need to fix it?